src/transforms/at-range.js

```diff
@@ -204,12 +204,17 @@ export function deleteBackwardAtRange(transform, range, n = 1, options = {}) {
     return
   }
 
-  // If the closest block is void, delete it.
   const block = document.getClosestBlock(startKey)
+  // If the closest block is void, delete it.
   if (block && block.isVoid) {
     transform.removeNodeByKey(block.key, { normalize })
     return
   }
+  // If the closest is not void, but empty, remove it
+  if (block && !block.isVoid && block.isEmpty) {
+    transform.removeNodeByKey(block.key, { normalize })
+    return
+  }
```

Comment:
Can you add this same (but reversed, for "next") logic to the `deleteForward` transform as well with a test? Thanks!!